src/api-reference/invoice/purchase-request-endpoints.markdown

```diff
@@ -3,26 +3,33 @@ title: Purchase Request
 layout: reference
 ---
 
-{% include prerelease.html %}
-
 # Purchase Request
-
 * [Create a new purchase request](#post)
 * [Get details of purchase request](#get)
 * [Schema](#create_purchase_request_schema)
-* [Response schema](#create_purchase_request_schema-response)
+  * [Create Purchase Request Schema](#create_purchase_request_schema)
+  * [Create Purchase Response Schema](#create_purchase_request-response-schema)
+  * [Get Purchase Request Response Schema](#get_purchase_request_response_schema)
+  * [Error](#error)
 * [Error codes](#error-codes)
 
-## Version
 
-4.0
+## Version
+4.0<br>
+**PRE-RELEASE:** The API is in Pre-Release and that only authorized early adopters should be using the API. Please contact <a href="mailto:charles.muellner@sap.com">charles.muellner@sap.com</a> for more information.
```

Comment:
This should be replaced with:

```
{% include prerelease.html %}
```

...and I'm fairly certain we only want team names or non-personal emails listed.

Reply:
Fixed